apps/examples/testcase/ta_tc/arastorage/utc/utc_arastorage_main.c

```diff
@@ -35,285 +36,262 @@
 #endif
 
 #define RELATION_NAME "rel"
-#define RELATION_INDEX "bplustree"
+#define INDEX_BPLUS "bplustree"
+#define INDEX_INLINE "inline"
 #define QUERY_LENGTH 128
 
 #define DATA_SET_NUM 10
+#define DATA_SET_MULTIPLIER 40
 /****************************************************************************
  *  Global Variables
  ****************************************************************************/
 
 static db_cursor_t *g_cursor;
 
-static int g_arastorage_tc_count;
-
-static int g_arastorage_tc_fail_count;
-
-
-const static char *g_attribute_set[] = {"id", "date", "fruit", "weight"};
+const static char *g_attribute_set[] = {"id", "date", "fruit", "value", "weight"};
 
 struct arastorage_data_type_s {
-	int int_value;
 	long long_value;
 	char *string_value;
 	double double_value;
 };
 
 typedef struct arastorage_data_type_s arastorage_data_t;
 
-const static arastorage_data_t g_arastorage_data_set[DATA_SET_NUM] = {
-	{ 1,        20160101,       "apple",           1.0001},
-	{ 2,        20160102,       "banana",          2.0002},
-	{ 3,        20160103,       "mango",           3.0003},
-	{ 4,        20160104,       "orange",          4.0004},
-	{ 5,        20160105,       "melon",           5.0005},
-	{ 6,        20160106,       "kiwi",            6.0006},
-	{ 7,        20160107,       "peach",           7.0007},
-	{ 8,        20160108,       "cherry",          8.0008},
-	{ 9,        20160109,       "strawberry",      9.0009},
-	{10,        20160110,       "watermelon",     10.0010}
+const static struct arastorage_data_type_s g_arastorage_data_set[DATA_SET_NUM] = {
```

Comment:
You make type definition, arastorage_data_t, but not used. Let's remove arastorage_data_t or change this to _t.